CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## v1.10.0
+
+<a name="breaking_changes_1.9.0">[Breaking Changes:](#breaking_changes_1.9.0)</a>
+
+- [plugin] added `createDeployQuickOpenItem` method to create `DeployQuickOpenItem` class in order to make extension deploy command extensible [#8919] (https://github.com/eclipse-theia/theia/pull/8919)
+
```

Comment:
@odeliat it is not a "breaking change" in your latest implementation, you can include the entry simply under `v.1.10.0` if you'd like it documented (I generally prepare the changelog before releases so there's no issue if you omit it).